authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerPaimonE2EIT.java

```diff
@@ -0,0 +1,235 @@
+            containerSuite.getHiveRangerContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+
+    generateRangerSparkSecurityXML();
+
+    sparkSession =
+        SparkSession.builder()
+            .master("local[1]")
+            .appName("Ranger Hive E2E integration test")
+            .config("spark.sql.catalog.paimon", "org.apache.paimon.spark.SparkCatalog")
+            .config("spark.sql.catalog.paimon.metastore", "hive")
+            .config("spark.sql.catalog.paimon.uri", HIVE_METASTORE_URIS)
+            .config(
+                "spark.sql.catalog.paimon.warehouse",
+                String.format(
+                    "hdfs://%s:%d/user/hive/warehouse",
+                    containerSuite.getHiveRangerContainer().getContainerIpAddress(),
+                    HiveContainer.HDFS_DEFAULTFS_PORT))
+            .config("spark.sql.catalog.paimon.cache-enabled", "false")
+            .config(
+                "spark.sql.extensions",
+                "org.apache.kyuubi.plugin.spark.authz.ranger.RangerSparkExtension,"
+                    + "org.apache.paimon.spark.extensions.PaimonSparkSessionExtensions")
+            .enableHiveSupport()
+            .getOrCreate();
+
+    createMetalake();
+    createCatalog();
+
+    RangerITEnv.cleanup();
+    metalake.addUser(System.getenv(HADOOP_USER_NAME));
+  }
+
+  @AfterAll
+  public void stop() {
+    cleanIT();
+  }
+
+  @Override
+  protected void useCatalog() throws InterruptedException {
+    String userName1 = System.getenv(HADOOP_USER_NAME);
+    String roleName = currentFunName();
+    SecurableObject securableObject =
+        SecurableObjects.ofMetalake(
+            metalakeName, Lists.newArrayList(Privileges.UseCatalog.allow()));
+    metalake.createRole(roleName, Collections.emptyMap(), Lists.newArrayList(securableObject));
+    metalake.grantRolesToUser(Lists.newArrayList(roleName), userName1);
+    waitForUpdatingPolicies();
+    sparkSession.sql(SQL_USE_CATALOG);
+    metalake.deleteRole(roleName);
+    waitForUpdatingPolicies();
+  }
+
+  @Override
+  protected void checkUpdateSQLWithReadWritePrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to update yet.
+  }
+
+  @Override
+  protected void checkUpdateSQLWithReadPrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to update yet.
+  }
+
+  @Override
+  protected void checkUpdateSQLWithWritePrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to update yet.
+  }
+
+  @Override
+  protected void checkDeleteSQLWithReadWritePrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to delete yet.
+  }
+
+  @Override
+  protected void checkDeleteSQLWithReadPrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to delete yet.
+  }
+
+  @Override
+  protected void checkDeleteSQLWithWritePrivileges() {
+    // Kyuubi Paimon Ranger plugin doesn't support to delete yet.
+  }
+
+  @Override
+  protected void checkHaveNoPrivileges() {
```

Comment:
Change function name to `checkWithoutPrivileges()`?

Reply:
OK.